lib/rules/balanced-braces.js

```diff
@@ -0,0 +1,61 @@
+'use strict';
+
+const types = require('../utils/types');
+const ember = require('../utils/ember');
+
+//------------------------------------------------------------------------------
+// General rule -  Braces has to be balanced.
+//------------------------------------------------------------------------------
+
+const ERROR_MESSAGE = 'Unbalanced braces';
+
+module.exports = {
+  meta: {
+    type: 'problem',
+    docs: {
+      description: 'require that braces to be balanced.',
+      category: 'Possible Errors',
+      recommended: true,
```

Comment:
New rules should not be added as recommended, but we will enable this rule as recommended in the next major release :)